brian2/devices/cpp_standalone/device.py

```diff
@@ -572,7 +564,12 @@ def check_openmp_compatible(self, nb_threads):
         if nb_threads > 0:
             logger.warn("OpenMP code is not yet well tested, and may be inaccurate.", "openmp", once=True)
             logger.diagnostic("Using OpenMP with %d threads " % nb_threads)
-    
+            if prefs.devices.cpp_standalone.openmp_spatialneuron_strategy != None:
```

Comment:
This should probably be ``is not`` rather than ``!=``?

Reply:
Agreed.